File: 은지/K번째수.py
def solution(array, commands):
    answer = []
    for j in range(len(commands)):
        # array 배열 자르기
        subA = []
        for i in range(len(array)):
            subA = array[commands[j][0]-1:commands[j][1]]
        # 자른 배열 정렬
        subA.sort()
        # commands에서 언급한 연산 결과
        answer.append(subA[commands[j][2]-1])
    return answer

File: 은지/K번째수.js
function solution(array, commands) {
    var answer = [];
    for (let j=0; j < commands.length; j++) {
      var sub_a = array.slice(commands[j][0]-1, commands[j][1])
      sub_a.sort((a, b) => a-b)
      answer.push(sub_a[commands[j][2]-1])
      sub_a = []
    }
    return answer;
}